platform-tests/src/test/kotlin/org/nd4j/samediff/frameworkimport/onnx/loader/TestOnnxProcessLoader.kt

```diff
@@ -37,12 +37,15 @@ class TestOnnxProcessLoader {
 
     @Test
     fun testMobileNet() {
+        System.setProperty("org.bytedeco.javacpp.nopointergc","true")
+        Nd4j.getExecutioner().enableDebugMode(true)
+        Nd4j.getExecutioner().enableVerboseMode(true)
         val importer = OnnxFrameworkImporter()
         val resource = ClassPathResource("mobilenet.onnx").file
         val inputs = Nd4j.ones(1,3,224,224)
         val inputDict = mapOf("input.1" to inputs)
         val runImport = importer.runImport(resource.absolutePath,inputDict)
-
+        println("Ran import")
```

Comment:
I guess this was left over from a debugging session?

Reply:
Yeah.